src/content/docs/tutorial-errors/solve-critical-errors.mdx

```diff
@@ -0,0 +1,105 @@
+
+<Steps>
+    <Step>
+
+    ### Open the Log details window
+
+Looking at our example, the APM agent has pulled logs related to `api-gateway`, giving us event data about our error:
+
+    <img
+        title="Overview errors affecting your services"
+        alt="A screenshot showing an app with many errors"
+        src={apmLogs}
+    />
+
+Our logs in context capability formats your logs information, though you have the option to investigate the unformatted log as well. 
+    </Step>
+    <Step>
+
+    ### Find the failure point in your system
+        
+Depending on the nature of the log, you may have more or less logging detail to sort through. Because you spent time upfront narrowing down the service closest to the failure and choosing the likely error group, you have time to read your logs. 
+
+<img
+    title="Overview errors affecting your services"
+    alt="A screenshot showing an app with many errors"
+    src={apmLogsZoom}
+    />    
+
+According to your logs, your problem is:
+
+* A timeout error: `error.error.code: ETIMEDOUT`
+* Related to your customer API: `error.endpoint: customers-api-internal`
+* Agnostic to the kind of request sent to the customer API: `error.request: /api/customers/search/Kirlin/-/`
+
+You have enough context to tie it together. You conclude that there's a dependency failure: All calls to the customer API are timing out and causing upstream requests from `api-gateway` to fail. 
+
+</Step>
+
+<Step>
+
+    ### Find the failure point in your code with stack traces
+
+When you run your code, New Relic captures your exceptions or runtime errors and organizes them into a waterfall view. These are your stack traces. Depending on the language used, these stack traces provide detailed messages and the locations of your failure points.  
+
+    <img
+    title="Overview errors affecting your services"
+    alt="A screenshot showing an app with many errors"
+    src={apmStackTraces}
+    />    
+
+While your logs tells you what's causing the error, your stack traces identify the where. You can use these stack traces to find exactly where in your code an error is occurring, then connect those errors to the owning team.
+
+    </Step>
+    <Step>
+
+### Open in your IDE from New Relic 
+
+To open your code from New Relic, you need to [install Codestream](docs/codestream/start-here/install-codestream). This is an optional but highly recommended feature that lets you open your IDE directly from stack traces:
+
+    <img
+    title="Overview errors affecting your services"
+    alt="A screenshot showing an app with many errors"
+    src={apmErrorIDE}
+    />  
+
+Once you've gotten the full context, it's time to assign that error to the correct team and update the error's status. 
```

Comment:
I would delete this ¶. It' sredundant with the next section, and it makes it seem like assigning the error is somehow tied in with CodeStream